Makefile

```diff
@@ -2,8 +2,8 @@
 
 CPUARCH=$(shell /usr/bin/arch)
 # this needs to be in sync with debian/changelog and build/build.common.proj
-BUILD_NUMBER="2.3.5"
-BUILD_VCS_NUMBER="0f07a86ab6e17cf779698c83231aa03250b89c6d"
+BUILD_NUMBER="2.3.8"
```

Comment:
That part I know. But then how do I account for this line:

<BUILD_NUMBER Condition="'$(BUILD_NUMBER)' == ''">2.3.5</BUILD_NUMBER>

and this file not being updated:


			<li>DEV_VERSION_NUMBER<ol>
				<li>L10NSharp: Improve debug information for linux localization</li>
			</ol></li>
			<li>2.3.2<ol>
				<li>FLEx Bridge: Enhance Windows installer</li>
			</ol></li>